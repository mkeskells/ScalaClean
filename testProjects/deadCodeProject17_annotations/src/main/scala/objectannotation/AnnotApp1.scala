package objectannotation

object AnnotApp1 extends App {
  UsedObject
}


@UsedAnnotation1(classOf[UsedByAnnotation1])
object UsedObject {
  def unused1 = ???
}

class UsedAnnotation1(value: Class[_])

class UsedByAnnotation1(value: Class[_])

class UnusedAnnotation(value: Class[_])
