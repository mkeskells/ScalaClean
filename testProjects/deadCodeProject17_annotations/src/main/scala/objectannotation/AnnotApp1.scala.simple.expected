package objectannotation

object AnnotApp1 extends App {
  UsedObject
}


@UsedAnnotation1(classOf[UsedByAnnotation1])
object UsedObject {
}

class UsedAnnotation1(value: Class[_])

class UsedByAnnotation1(value: Class[_])
